v2/jdbc-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/options/JdbcToBigQueryOptions.java

```diff
@@ -54,8 +54,11 @@ public interface JdbcToBigQueryOptions
       },
       groupName = "Source",
       description = "JDBC connection URL string.",
-      helpText = "The JDBC connection URL string. You can pass in this value as a string that's encrypted with a Cloud KMS key and then Base64-encoded. Remove whitespace characters from the Base64-encoded string.",
-      example = "jdbc:oracle:thin:@some-host:port:sid")
```

Comment:
@damccorm 
This file is used in both MySQL to BQ and Oracle to BQ templates. 
So example is conflicting, should we make it generic like <mysql/oracle> ? OR 
Specify both examples in it mentioning For Oracle, For MySQL.

Reply:
Lets leave it as is. I think it should be obvious that the example needs modified